src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,9 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.toLowerCase();
+        text = text.replaceAll("\\W", "");
+        StringBuilder revers = new StringBuilder(text);
```

Comment:
can we squash these three lines in the one line?

Reply:
yes, we can.